components/schemas/environments/services/SchedulerEnvironmentService.yml

```diff
@@ -0,0 +1,28 @@
+---
+title: SchedulerEnvironmentService
+type: object
+nullable: true
+description: Information about the environments scheduler service(s).
```

Comment:
service. can't run in HA

Reply:
Irrelevant here though.  It still has the HA property, it will just always be false

Reply:
yes but i want to convey via description that it will NEVER be HA even if the property exist.